app/api/v_sphere/virtual_machine.rb

```diff
@@ -134,6 +134,10 @@ def ==(other)
       equal? other
     end
 
+    def macs
```

Comment:
@MaxSchneider1337 It might have (@regnujAx and I discussed this yesterday).
I will ask the customer on Wednesday and then we will see whether we have to change something.